lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -105,7 +105,7 @@ sub find_by_area {
     my (
         $containment_query,
         @containment_query_args,
-    ) = get_area_containment_query('$2', 'any(array[area, begin_area, end_area])');
+    ) = get_area_containment_query('$2', 'any(array[area, begin_area, end_area])', 1);
```

Comment:
Changes look good, but it might improve clarity slightly if we made this a keyword arg like `, check_all_levels => 1)`.